seed/supabase-twitter-clone/README.md

```diff
@@ -60,7 +60,6 @@ First, let's set up a local development environment for the Supabase Twitter clo
 
     ```bash
     npx supabase login
-    npx supabase init
```

Comment:
I got this when I ran this:

```
 failed to create config file: open supabase/config.toml: file exists
Try rerunning the command with --debug to troubleshoot the error.
```

I guess we don't need to `init` anymore since we already checked in the stuff that was generated during init.